task_1/anna.zinovatnaya/src/main/java/task_1/Dialog.java

```diff
@@ -0,0 +1,272 @@
+package main.java;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Vector;
+
+public class Dialog {
+
+  /*
+  Files and Folders represent real files that are under git. If two files are equal they are still stored two times
+   */
+  private Folder currentFolder;
+  private Folder rootFolder; // modified area
+
+  /*
+  Map stores committed elements (files, folders and commits). If two elements are equal, they are stored only once.
+   */
+  private Map<String, String> commitedFilesMap; // committed area
+
+  /*
+  Vector stores history of commits
+  */
+  private Vector<Commit> commits;
+
+
+
+  public Dialog() {
+    this.currentFolder = new Folder();
+    this.rootFolder = currentFolder;
+    this.commitedFilesMap = new HashMap<>();
+    this.commits = new Vector<>();
+  }
+
+
+
+  void start() {
+
+    String option = "0";
+
+    Scanner scanner = new Scanner(System.in, "UTF-8");
+
+    System.out.println("Commands:");
```

Comment:
Move all the system outs in some private method, such as **showMenuOptions** or something like that. Code will become clearer.